src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/Error/SqlExtendedQueryTagErrorStoreV2.cs

```diff
@@ -0,0 +1,14 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag.Error
+{
+    internal class SqlExtendedQueryTagErrorStoreV2 : SqlExtendedQueryTagErrorStoreV1
```

Comment:
Instead of implementing empty classes for every version, which will get more in the future, can we just handle version not supported when accessing a service for a specific version that is not found?

Reply:
As this is an issue for many other of our stores, such as instance and extended query tags, I have opened a [user story](https://microsofthealth.visualstudio.com/Health/_boards/board/t/Medical%20Imaging/Stories/?workitem=84235) to work on this in a separate PR